core/src/main/java/org/apache/iceberg/PartitionsTable.java

```diff
@@ -82,7 +82,12 @@ public class PartitionsTable extends BaseMetadataTable {
                 10,
                 "last_updated_snapshot_id",
                 Types.LongType.get(),
-                "Id of snapshot that last updated this partition"));
+                "Id of snapshot that last updated this partition"),
+            Types.NestedField.required(
+                11,
+                "total_data_size_in_bytes",
```

Comment:
For partition stats, I am using the field name as `data_file_size_in_bytes` here.

https://github.com/apache/iceberg/pull/7105/files#diff-36347a47c3bf67ea2ef6309ea96201814032d21bb5f162dfae4045508c15588aR736

Let us wait and see what others think about the naming of this field. Based on that either you or me have to modify accordingly.

Reply:
Ah good point, I'm ok with total_data_file_size_in_bytes then to be consistent, initially thought its a bit long.

Reply:
I am also ok with `total_data_file_size_in_bytes`